### ISSUE TITLE
Codeception 5.0.0-RC4 fails without a clear cause (for me)

### ISSUE THREAD

**Issue by wazum** [infection/codeception-adapter#18]
Maybe someone can help me here:

All tests pass.

```
Infection - PHP Mutation Testing Framework version 0.26.12


Running initial test suite...

codeception version: 5.0.0-RC4

    3 [============================] < 1 sec
 [ERROR] Project tests must be in a passing state before running Infection.
         Check the executed command to identify the problem: '/app/vendor/bin/codecept' 'run' '--debug' '--no-colors'
         '--fail-fast' '--coverage-phpunit' 'codeception-coverage-xml' '--xml' '/tmp/infection/junit.xml' '-o' 'paths:
         output: /tmp/infection' '-o' 'coverage: enabled: true' '-o' 'coverage: include: [src/*.php]' '-o' 'settings:
         shuffle: true'
         codeception reported an exit code of 143.
         Refer to the codeception's output below:
         STDOUT:
         Codeception PHP Testing Framework v5.0.0-RC4 https://helpukrainewin.org
         Powered by PHPUnit 9.5.21 #StandWithUkraine
         [Seed] 1333019323

         STDERR:
```

When I run the command I get

```
XDEBUG_MODE=coverage /app/vendor/bin/codecept run --debug --no-colors --fail-fast --coverage-phpunit 'codeception-coverage-xml' --xml '/tmp/infection/junit.xml' -o 'paths: output: /tmp/infection' -o 'coverage: enabled: true' -o 'cov
erage: include: [src/*.php]' -o 'settings: shuffle: true'
Codeception PHP Testing Framework v5.0.0-RC4 https://helpukrainewin.org
Powered by PHPUnit 9.5.21 #StandWithUkraine
[Seed] 241093037

  Printing JUNIT XML report to /tmp/infection/junit.xml

App\Tests.acceptance Tests (0) --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Modules: PhpBrowser, \App\Tests\Helper\Acceptance
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

In LocalServer.php line 163:

  [RuntimeException]
  Code coverage file /tmp/infection/c3tmp/codecoverage.serialized does not exist
```

Any hints what the problem might be? tia

**Comment by wazum** [comment]
Tried it with a pre-generated coverage too
```
 ./vendor/bin/infection --test-framework=codeception --coverage=./tests/_output/coverage-phpunit/
```
but Codeception seems to be called with the parameters to output coverage nonetheless.

**Comment by wazum** [comment]
I found the problem, I had to set the environment variable
```
export XDEBUG_MODE=coverage
```

**Comment by maks-rafalko** [comment]
Hello,

we just released [0.26.13](https://github.com/infection/infection/releases/tag/0.26.13) with a fix to automatically add `XDEBUG_MODE`, probably worth upgrading and test without `export`ing it manually.
